front/main/app/mixins/discussion-moderation-model.js

```
@@ -164,6 +164,54 @@ export default Ember.Mixin.create({
 		});
 	},
 
+
+	/**
+	 * Locks a post in the service
+	 * @param {object} post
+	 * @returns {Ember.RSVP.Promise|void}
+	 */
+	lockPost(post) {
+		if (checkPermissions(post, 'canDelete')) {
```

do you plan to do something if that is false?
otherwise it'll silently ignore it

jQuery does that - and I don't think we are super fans of that. What do you think?